mlx-mkbfb

```diff
@@ -73,8 +73,9 @@ image_list = (
     ("trusted-key-cert", "Trusted key certificate", 7),
     ("bl31-key-cert", "EL3 Runtime Firmware BL3-1 key certificate", 9),
     ("bl31-cert", "EL3 Runtime Firmware BL3-1 content certificate", 13),
-    ("bl31", "EL3 Runtime Firmware BL3-1 bin", 3),
+    ("psc-certs", "PSC attestation certificates",          53),
```

Comment:
Question: based on this configuration the certificates are sent to the PSC after the DDR training, right?

Could you please remind me why we would like to send the certificates after the DDR training and not before?

Reply:
> Question: based on this configuration the certificates are sent to the PSC after the DDR training, right?
> 
> Could you please remind me why we would like to send the certificates after the DDR training and not before?

There are two reasons...
1. pscfw might be not ready at that time. Doing it before DDR training will involves some delay/wait.
2. I remembered there are some discussion on before, can't remember exactly. Let me try to find some note...

Reply:
Thank you Liming.

No worries. Let's push the changes and later on we can re-order the list as needed :)